apps/pipeline/lib/pipeline/writer/table_writer.ex

```diff
@@ -67,20 +68,23 @@ defmodule Pipeline.Writer.TableWriter do
     |> Compaction.complete(table)
   end
 
+  @impl Pipeline.Writer
+  @spec delete(dataset: [term()]) :: :ok | {:error, term()}
+  def delete(args) do
+    table_name =
+      Keyword.fetch!(args, :dataset)
+      |> StatementUtils.parse_table_name()
+
+    table_name
+    |> StatementUtils.create_new_table_with_existing_table()
+
+    StatementUtils.drop_table(table_name)
```

Comment:
Is it possible to just rename the table, rather than creating a new table and dropping the old one?

Reply:
Understand the reasoning now... thank you.